Authy2FA/Controllers/ManageController.cs

```diff
@@ -127,7 +127,7 @@ public ActionResult AddPhoneNumber()
                 user.AuthyId = result.UserId;

```

Comment:
Probably should wrap this and the following lines of code in a null reference check.   The first time I ran this I forgot to drop my authy key into the config file so the result was null, and calling user.AuthyId caused a null ref exception.  Seems like we could detect the failed call to Authy and throw a more helpful exception.